lib/rps.rb

```diff
@@ -0,0 +1,25 @@
+class RPS
+
+  attr_reader :player, :opponent
+
+  def initialize(player, opponent)
+    @player = player.downcase
+    @opponent = opponent
+  end
+
+  def result
+    return "draw" if draw? == true
+    player_wins? == true ? "win" : "lose"
```

Comment:
Not necessary, but I think this could be refactored one step further to: player_wins? ? "win" : "lose"